Source/WebCore/platform/graphics/displaylists/DisplayListItems.h

```diff
@@ -981,6 +982,35 @@ class DrawDotsForDocumentMarker {
     bool m_styleShouldUseDarkAppearance { false };
 };
 
+template<class Encoder>
+void DrawDotsForDocumentMarker::encode(Encoder& encoder) const
```

Comment:
Is this just work that still needs to be done?  It's not obvious why this is being added.

Reply:
Yes, this needs to be ported but I didn't want to bloat this PR too much. Was going to put together a big DisplayList change at some point.